src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,70 @@
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+interface Props {
+  key: string;
+  person: Person;
+  mother: Person | null;
+  father: Person | null;
+}
+
+export const PersonInfo: React.FC <Props> = ({
+  key,
+  person,
+  mother,
+  father,
+}) => {
+  const {
+    name,
+    sex,
+    born,
+    died,
+    fatherName = '-',
+    motherName = '-',
+    slug,
+  } = person;
+
+  const { slectedPersonSlug = '' } = useParams();
```

Comment:
Be careful when naming, there is a typo in variable name `slectedPersonSlug`. It should be `selectedPersonSlug`.